internal/elasticsearch/ingest/datastream.go

```diff
@@ -283,10 +286,11 @@ func putIngestPipeline(api *elasticsearch.API, pipeline Pipeline) error {
 	return nil
 }
 
-func getIngestPipeline(api *elasticsearch.API, pipeline Pipeline) error {
-	r, err := api.Ingest.GetPipeline(func(request *elasticsearch.IngestGetPipelineRequest) {
-		request.PipelineID = pipeline.Name
-	})
```

Comment:
This can be replaced by 
```golang
// WithPipelineID - comma separated list of pipeline ids. wildcards supported.
func (f IngestGetPipeline) WithPipelineID(v string) func(*IngestGetPipelineRequest) {
    return func(r *IngestGetPipelineRequest) {
        r.PipelineID = v
    }
}
```